homeassistant/components/lifx/light.py

```diff
@@ -616,6 +635,22 @@ def effect(self):
             return f"lifx_effect_{effect.name}"
         return None
 
+    @property
+    def extra_state_attributes(self):
+        """Return extra state attributes for certain bulb types."""
+        if lifx_features(self.bulb)["hev"] is True:
+            if self.bulb.hev_cycle.get("remaining", 0) > 0:
+                return {
+                    "hev_cycle_active": True,
+                    "hev_cycle_duration": self.bulb.hev_cycle["duration"],
+                    "hev_cycle_remaining": self.bulb.hev_cycle["remaining"],
+                    "hev_restore_power": self.bulb.hev_cycle["last_power"],
```

Comment:
These probably shouldn't be attributes? Why not make these actual sensors?

Additionally, things like "remaining" is the number of seconds remaining? If that is the case, this will cause a lot of state updates. For those things, it is better to use provide a datetime of when it's expected to finish.